lib/schema/schema.sql

```diff
@@ -0,0 +1,18 @@
+-- Books table
+CREATE TABLE books (
+    id                  SERIAL PRIMARY KEY,
+    publish_date        DATE,
+    archived            BOOLEAN,
+    genre_id            INT REFERENCES genre(id)
+    author_id           INT REFERENCES author(id)
+    source_id           INT REFERENCES source(id)
+    label_id            INT REFERENCES label(id)
+);
+
+-- Label table
+CREATE TABLE label (
+    id                  SERIAL PRIMARY KEY,
+    title               VARCHAR(100),
+    color               VARCHAR(100),
+    items               JSON
```

Comment:
This is a 'many-to-one' relationship between each item table and the `label` table. I think the most cleaner way to translate so in SQL is to add a `FK` in each item table referencing on `label` table. That said I suggest you remove the `items` column from `label`